packages/react-native/ReactAndroid/src/main/java/com/facebook/react/views/scroll/ReactScrollViewAccessibilityDelegate.java

```diff
@@ -65,10 +65,6 @@ private void onInitializeAccessibilityEventInternal(View view, AccessibilityEven
       Integer firstVisibleIndex = null;
       Integer lastVisibleIndex = null;
 
-      if (!(contentView instanceof ViewGroup)) {
```

Comment:
This does look like a change in behaviour - won't the type casts below throw if this condition is unmet?

Reply:
You're totally right. I restored the original condicional